diff --git a/node_modules/three-stdlib/.bun-tag-3258d11d4bf116b b/.bun-tag-3258d11d4bf116b
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/three-stdlib/.bun-tag-e7c08fad3cb3ecb0 b/.bun-tag-e7c08fad3cb3ecb0
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/effects/AsciiEffect.js b/effects/AsciiEffect.js
index 3947faaff4feb599f96c7b87615a2e1ab267d541..6123a0c8b1dbcb2e4b22018a7fad6531ae7bf3a8 100644
--- a/effects/AsciiEffect.js
+++ b/effects/AsciiEffect.js
@@ -86,7 +86,7 @@ const AsciiEffect = function (renderer, charSet, options) {
 
   const aDefaultCharList = ' .,:;i1tfLCG08@'.split('');
   const aDefaultColorCharList = ' CGO08@'.split('');
-  var strFont = 'courier new, monospace';
+  var strFont = 'var(--font-mono)';
   const oCanvasImg = renderer.domElement;
   var oCanvas = document.createElement('canvas');
 
